packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/HeaderActionButton.tsx

```diff
@@ -1,7 +1,7 @@
 import { Button, ButtonProps, IconButton, IconButtonProps } from '@trezor/components';
+import { breakpointThresholds } from '@trezor/styles';
 
-import { useSelector } from 'src/hooks/suite';
-import { selectWindowSize } from 'src/reducers/suite/windowReducer';
+import { useResponsiveContext } from '../../../../../support/suite/ResponsiveContext';
```

Comment:
Refactor tip:

`ResponsiveContextProvider` has a prop `sidebarWidthFromRedux={sidebarWidthFromRedux}`

This prop is pulled from Redux in `SuiteLayout`.

Which means that anytime the sidebarWidth is cahnged, the whole layout re-renderes.

I'd put the `const sidebarWidthFromRedux = useSelector(state => state.suite.settings.sidebarWidth);`

In `ResponsiveContextProvider` to solve this.